sc2reader/resources.py

```diff
@@ -270,9 +270,16 @@ def __init__(self, replay_file, filename=None, load_level=4, engine=sc2reader.en
         # Load basic details if requested
         if load_level >= 1:
             self.load_level = 1
-            for data_file in ['replay.initData', 'replay.details', 'replay.attributes.events']:
+            files = [
+                'replay.initData.backup',
```

Comment:
do you happen to know if all replay files have these `replay.initData.backup`, or is it just these special ones from blizzard? either way, can you comment in why we're doing this?

Reply:
Added a comment.